What to do with `additional materials` (#132)
formal-methods-mpi/Workshop.jl: docs/src/5_additional/additional-materials.md

# Additional Materials

> This chapter contains some additional materials and exercises.

Reviewer: ```suggestion
This chapter contains some additional materials and useful links to explore.
```

You should do it only after you finished the rest of the workshop.

> You can do them in any order, just pick something you are interested in.

Reviewer: ```suggestion
```

> - [Automatic Differentiation](@ref) contains an advanced exercise of building an automatic differentiation system from scratch.

Reviewer: ```suggestion
If you like a challenge, we have prepared a more advanced exercise, [Automatic Differentiation](@ref).
```

> 

Reviewer: ```suggestion
But you can also just check out the following links and see if you find something of interest.
```
